server/map_rpc_server_test.go

```diff
@@ -336,6 +336,63 @@ func TestGetSignedMapRootByRevision(t *testing.T) {
 	}
 }
 
+func TestSetLeavesEmpty(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+	ctx := context.Background()
+
+	root := &trillian.SignedMapRoot{
+		MapRoot:   []byte("\000\001 \306h\237\020\201*\t\200\227m\2253\3308u(!f\025\225g\3545\025W\026\301A:\365=j\025\315\333#\364\266\341\026\000\000\000\000\000\000\000\000\000\000"),
+		Signature: []byte("0E\002!\000\301\332Z\324k\334\354\270n\023n\"\030\242rz;\310\"9\331 \325\375\205W\305\241\014\337Z\335\002 |VJ\026T\352l?\203\354\340\222\330#\273y22\354U\307_x\376\3771E\374\315h\371\307"),
+	}
+
+	for _, tc := range []struct {
+		root     *trillian.SignedMapRoot
+		revShift int64
+		smrErr   error
+		wantErr  bool
+	}{
+		{root: root, wantErr: false},
+		{root: root, revShift: -1, wantErr: true},
+		{root: root, revShift: -10, wantErr: true},
+		{root: root, revShift: 1, wantErr: true},
+		{root: nil, wantErr: true},
+		{root: &trillian.SignedMapRoot{MapRoot: []byte("invalid")}, wantErr: true},
+		{root: nil, smrErr: errors.New("you shall not pass"), wantErr: true},
+	} {
+		t.Run("", func(t *testing.T) {
```

Comment:
The empty test description here seems suboptimal. On failure is it easy to identify which of the cases failed?

Reply:
Yeah, the framework auto-adds a suffix looking something like `#01`, which is logically equivalent to manually adding `strconv.Itoa(i)`. I just find this description bit unnecessary, the index should be enough.